app/models/spotlight/resources/dor_resource.rb

```diff
@@ -36,7 +36,12 @@ def resource_to_solr base_doc = {}, r
     end
     
     def existing_solr_doc_hash doc_hash
-      exhibit.blacklight_config.solr_document_model.new(doc_hash).to_solr
+      # if not in the context of an exhibit, just continue without that data
+      if exhibit
+        exhibit.blacklight_config.solr_document_model.new(doc_hash).to_solr
+      else
+        doc_hash
```

Comment:
I think this can return an empty hash instead of the `doc_hash` parameter, no? It looks like it gets merged into the doc hash again on line 35.